public/editor/scripts/main.js

```diff
@@ -52,12 +53,14 @@ require(["jquery", "bowser"], function($, bowser) {
 
   Bramble.once("error", onError);
 
-  function init(BrambleEditor, Project, SSOOverride, ProjectRenameUtility) {
+  function init(BrambleEditor, Project, SSOOverride, ProjectRenameUtility, unloadProtection) {
```

Comment:
@aayushsanghavi I think what @Pomax meant (I might be wrong), is that the naming should be consistent throughout this file.

Short explanation: it shouldn't be `unloadProtection`, it should be `reloadProtection` here

Longer explanation:
The parameter that is passed here in this function (what is currently called `unloadProtection`) might not be the right name to use. What you are actually getting in this parameter, is the defined module itself (and from looking at the `reload-protection.js` file, it looks like it's an object) and not the function you have defined in that module (`unloadProtection`). So, since the module is an object, and you named your module `reload-protection.js`, it makes more sense for this function parameter to be called `reloadProtection`. `unloadProtection` is an internal method in that module that you use in the module file; but in this file, we don't need to worry about that because it doesn't even have access to the `unloadProtection` function, only to the object that you returned.